flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/TestingAbstractInvokables.java

```diff
@@ -82,4 +84,94 @@ public void invoke() throws Exception {
 			}
 		}
 	}
+
+	/**
+	 * Test invokable which blocks after invoke was called.
+	 */
+	public static class BlockingInvokable extends AbstractInvokable {
```

Comment:
We already have `org.apache.flink.runtime.testtasks.BlockingNoOpInvokable` which is identical of this class. Could you just use it instead of introduce a new identical class?

Reply:
Thanks, will do that.